homeassistant/components/isy994/services.py

```diff
@@ -195,11 +196,36 @@ async def async_system_query_service_handler(service: ServiceCall) -> None:
                     isy.configuration["uuid"],
                 )
                 await isy.query(address)
+
+                # DEPRECATED NOTIFICATION
+                entity_registry = er.async_get(hass)
+                entries_for_this_config = er.async_entries_for_config_entry(
+                    entity_registry, config_entry_id
+                )
+                alternate_target = next(
+                    entity.entity_id
+                    for entity in entries_for_this_config
+                    if f"{address}_query" in entity.unique_id
+                )
```

Comment:
```suggestion
                entity_registry = er.async_get(hass)
                alternate_target = entity_registry.async_get_entity_id((Platform.BUTTON, DOMAIN, f"{address}_query"))
```

Untested but I think this should work

Reply:
Adjusted

Reply:
Works. 

```shell
2023-01-07 00:15:09.513 WARNING (MainThread) [homeassistant.components.isy994] The "isy994.system_query" service is deprecated and will be removed in 2023.5.0; use the "button.press" service and pass it a target entity ID of "button.office_crystal_lamp_2e_b8_75_query"
```

Reply:
On another note--you just showed me how I'll be able to eventually move the `isy994_control` events out of the entity logic and back up to the config level.